### PR TITLE
Add basic test for the plugin

### LINKED ISSUE #2: No tests / Tests missing
Hi,
  this plugin has no test at all, this is a situation that should be amended soon, so for example changes in both the plugin or the logstash core himself, are properly tested in the CI environment.

If you are giving this plugin some test, feel free to update https://github.com/elastic/logstash/issues/3740 so we keep track of it.

### PR DESCRIPTION
**Pull request by purbon**
Fixes #2

### CODE REVIEW

File: logstash-output-ganglia.gemspec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-ganglia'
-  s.version         = '1.0.0'
+  s.version         = '1.1.0'
```
Comment (ph) [inline logstash-output-ganglia.gemspec:4]: 1.1.0 -> 1.0.1

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# 1.1.0
```
Comment (ph) [inline CHANGELOG.md:1]: 1.1.0 -> 1.0.1

**Comment by ph** [comment]
Minor comments concerning the version of this plugin, after address it LGTM and the test are running.

**Comment by purbon** [comment]
@ph please see https://github.com/logstash-plugins/logstash-input-zeromq/pull/8 for more detials on this. More than welcome to be educated.

**Pull request merged**